draft-ietf-mls-protocol.md

```diff
@@ -818,35 +818,21 @@ strings with length Hash.length to HPKE key pairs.
 Ciphersuites are represented with the CipherSuite type. HPKE public keys
 are opaque values in a format defined by the underlying Diffie-Hellman
 protocol (see the Ciphersuites section of the HPKE specification for more
-information):
+information).
 
 ~~~~~
-enum {
-    P256_SHA256_AES128GCM(0x0000),
-    X25519_SHA256_AES128GCM(0x0001),
-    (0xFFFF)
-} CipherSuite;
-
 opaque HPKEPublicKey<1..2^16-1>;
 ~~~~~
 
-### Curve25519, SHA-256, and AES-128-GCM
-
-This ciphersuite uses the following primitives:
-
-* Hash function: SHA-256
-* AEAD: AES-128-GCM
+The ciphersuites are defined in section {{MLS Ciphersuites}}.
 
-When HPKE is used with this ciphersuite, it uses the following
-algorithms:
+### Notes on Diffie-Hellman groups
```

Comment:
I would make this section at the same level as "Ciphersuites" (so "## Diffie-Hellman Groups") and add explainer text that DH groups are used to build KEMs with HPKE

Reply:
I'll add the explainer, but I think it makes sense to keep it at this level, since it is about ciphersuites after all.

Reply:
(added)